README.md

```diff
@@ -34,6 +37,18 @@ database.  If that is successful you can run the app server with:
 
     $ foreman start
 
+### Getting Started (docker)
+
+* Install [Docker Compose](https://docs.docker.com/compose/install/)
+  * On non-linux systems this will involve installing Docker Toolbox and
+    starting a Docker Machine VM.
```

Comment:
What do you think about linking to the Docker Toolbox download page? https://www.docker.com/products/docker-toolbox